### PR TITLE
Add missing "exclude" option to README.MD

### PR DESCRIPTION
**Pull request by NadeenUdantha**

### CODE REVIEW

**Comment by sanex3339** [comment]
This option is CLI-only option, so it shouldn't be here. It's already described in the list of CLI-only options.
But I think that it's a good idea to add an additional list of CLI-options descriptions

File: README.md [hunk 1]
```diff
@@ -360,6 +360,7 @@ Following options are available for the JS Obfuscator:
     debugProtectionInterval: false,
     disableConsoleOutput: false,
     domainLock: [],
+    exclude: [],
```
Comment (sanex3339) [inline README.md:363]: This option is CLI-only option, so it shouldn't be here. It's already described in the list of CLI-only options.

Comment (sanex3339) [reply README.md:363]: But I think that it's a good idea to add an additional list of CLI-options descriptions

**Review by sanex3339 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NadeenUdantha) [reply README.md:363]: But it seems to be working fine with config js file 😐

**Pull request closed without merging**